src/main/java/org/broadinstitute/consent/http/enumeration/EmailType.java

```diff
@@ -22,7 +22,10 @@ public enum EmailType {
   DATASET_DENIED(18, "dataset-denied.html"),
   DATASET_APPROVED(19, "dataset-approved.html"),
   DAR_EXPIRED(20, "dar-expired.html"),
-  DAR_EXPIRATION_REMINDER(21, "dar-expiration-reminder.html");
+  DAR_EXPIRATION_REMINDER(21, "dar-expiration-reminder.html"),
+  NEW_PROGRESS_REPORT(22, "new-progress-report-request.html"),
```

Comment:
I would be confused seeing this enum without context - can we rename it to more closely match the function?
```suggestion
  NEW_PROGRESS_REPORT_REQUEST(22, "new-progress-report-request.html"),
```